train.py

```diff
@@ -126,6 +126,7 @@ def train(config):
     loggers.append(tb_logger)
 
     if config.get("wandb", False):
+        run = wandb.init()
```

Comment:
Sorry, but I'm not sure if it will work as I expected or I do not understand the code. I've read https://lightning.ai/docs/pytorch/stable/_modules/lightning/pytorch/loggers/wandb.html#WandbLogger and IMHO correct way of initializing wandb run and WandbLogger is:
* first create WandbLogger
* then `run = wandb.run`
In current version `run` will not have `project`, `name` set, because `WandbLogger` will use already created `run` (which will have probably random attributes).